### PR TITLE
add remote styles

### PR DESCRIPTION
**Pull request by SaraVieira**
- Adds possibility to pass an array of remote styles to be added to the CSS 

Need help with error handling and wondering if we should use the async version? I used this because it was the quickest since the async would need a bunch of changes in making everything be awaitable

example oranda.json for test:

```
{
  "theme": "axo",
  "remote_styles": ["https://unpkg.com/awsm.css@3.0.7/dist/awsm.min.css"]
}
```

related https://github.com/axodotdev/oranda/issues/24

### CODE REVIEW

**Review by ashleygwilliams — APPROVED** [review]

**Comment by SaraVieira** [comment]
@ashleygwilliams Requested a review again because I moved the css function to its own file as it was getting quite chunky

File: src/site/css/mod.rs
Comment (SaraVieira) [inline src/site/css/mod.rs]: In the idea of error fast I also added this

Comment (ashleygwilliams) [reply src/site/css/mod.rs]: i think you have a spelling error here: "Addicional" -> "Additional"

File: src/config/project/javascript.rs
Comment (ashleygwilliams) [inline src/config/project/javascript.rs]: can you say more about why we need this? in general i'd like to not include these

File: src/config/project/rust.rs
Comment (ashleygwilliams) [inline src/config/project/rust.rs]: same here

File: src/site/css/mod.rs
Comment (ashleygwilliams) [inline src/site/css/mod.rs]: since this is in a mod called "css" i would probably just call this "build" but this is a very tiny nitpick and is up to you to decide

File: src/site/css/mod.rs
Comment (ashleygwilliams) [inline src/site/css/mod.rs]: i would potentially break out some of these conditionals into smaller named functions to make it easier to test/read individual functionality, don't need to block on this tho, just sharing a stylistic pref

**Review by ashleygwilliams — COMMENTED** [review]
a few small things- once the spelling error is fixed this can be merged, everything else was more discussing styles

File: src/site/css/mod.rs
Comment (SaraVieira) [reply src/site/css/mod.rs]: Fixed

File: src/config/project/javascript.rs
Comment (SaraVieira) [reply src/config/project/javascript.rs]: oh, this is something I noticed when I downloaded the project, these imports would always give me warnings

File: src/site/css/mod.rs
Comment (SaraVieira) [reply src/site/css/mod.rs]: sounds good and done!

File: src/site/css/mod.rs
Comment (SaraVieira) [reply src/site/css/mod.rs]: broke both if statments into smaller functions

File: src/config/project/javascript.rs
Comment (ashleygwilliams) [reply src/config/project/javascript.rs]: oh huh- i'll file an issue to review them, we probably want to fix the import instead of just allowing unused

**Review by ashleygwilliams — APPROVED** [review]
🚀

**Comment by SaraVieira** [comment]
Removed the allow unused_imports so we remeber its there and will merge when ci passes

**Pull request merged**
